### DISCUSSION TITLE
Image don't show up

### DISCUSSION THREAD

**Discussion by spaceorcastudio** [sddebz/stable-diffusion-krita-plugin#36] [Q&A]
Hello! I installed the server and it's working on the browser, but when I run it on krita I just get a blank layer...
I thought it was because I had a old version of krita (5.0) but when I updated it to 5.1.1, I still get a blank layer...
![image](https://user-images.githubusercontent.com/86327732/191280472-88e04f26-99f0-489a-a6c3-5944b97178ec.png)

**Comment by yhslai** [comment]
Yeah I got blank image too. The webui works fine in my browser.

**Comment by yhslai** [comment]
Well, I found the reason. It only supports srgb. If your file isn't in srgb, it doesn't work at all.
